.github/workflows/tests_gpu.yml

```diff
@@ -0,0 +1,303 @@
+          module use /opt/modules
+          module load cuda/12.2
+          echo "${PATH}" >> $GITHUB_PATH
+          echo "LD_LIBRARY_PATH=${LD_LIBRARY_PATH}" >> $GITHUB_ENV
+          nvcc --version
+          nvidia-smi
+
+      - name: Clone Kokkos repository
+        # if: steps.kokkos-cache.outputs.cache-hit != 'true'
+        run: |
+          rm -rf kokkos_${{ matrix.exec_model }}
+          # git clone --branch fix-rtld-deepbind https://github.com/cz4rs/kokkos.git kokkos_${{ matrix.exec_model }}
+          # git clone --branch ${{ matrix.kokkos_version }} https://github.com/kokkos/kokkos.git kokkos_${{ matrix.exec_model }}
+          git clone --branch develop https://github.com/kokkos/kokkos.git kokkos_${{ matrix.exec_model }}
+          cd kokkos_${{ matrix.exec_model }}
+          git checkout e542e989af99cb87881ee2a0d06916541ee76006
+          git log | head
+          cd -
+          pushd . &> /dev/null
+
+      - name: Build Kokkos core library (Ubuntu)
+        # if: steps.kokkos-cache.outputs.cache-hit != 'true'
+        run: |
+          rm -rf ${{ github.workspace }}/Kokkos_install/${{ matrix.exec_model }}
+          mkdir -p ${{ github.workspace }}/Kokkos_install/${{ matrix.exec_model }}
+          cd kokkos_${{ matrix.exec_model }}
+          cmake -BBuild . -DCMAKE_INSTALL_PREFIX=${{ github.workspace }}/Kokkos_install/${{ matrix.exec_model }} \
+                          -DKokkos_ENABLE_COMPLEX_ALIGN=OFF \
+                          -DKokkos_ENABLE_SERIAL=ON \
+                          -DKokkos_ENABLE_${{ matrix.exec_model }}=ON \
+                          -DKokkos_ENABLE_DEPRECATION_WARNINGS=OFF \
+                          -DCMAKE_CXX_COMPILER="$(which g++-$GCC_VERSION)" \
+                          -DCMAKE_CXX_STANDARD=20 \
+                          -DCMAKE_POSITION_INDEPENDENT_CODE=ON \
+                          -G Ninja
+          cmake --build ./Build --verbose
+          cmake --install ./Build
+          cd -
+
+      - name: Test Kokkos core library (Ubuntu)
+        env: 
+          CMAKE_PREFIX_PATH: ${{ github.workspace }}/Kokkos_install/${{ matrix.exec_model }}
+        run: |
+          set -x
+          sudo apt-get update && sudo apt-get -y -q install gfortran-$GCC_VERSION
+          cd kokkos_${{ matrix.exec_model }}/example/build_cmake_installed
+          which -a gfortran || which -a gfortran-$GCC_VERSION
+          cmake -B build -DCMAKE_Fortran_COMPILER=gfortran-$GCC_VERSION
+          cmake --build build
+          cmake --build build --target test
+          set +x
+
+      - name: Cache installation directories
+        id: kokkos-cache
+        uses: actions/cache/save@v3
+        with:
+          key: ${{ matrix.os }}-kokkos${{ matrix.kokkos_version }}-${{ matrix.exec_model }}-${{ github.ref }}-${{ github.sha }}
+          path: ${{ github.workspace }}/Kokkos_install/${{ matrix.exec_model }}
+
+  cpptestswithKokkos:
+    if: ${{ github.sha == 'xxxxxxxx' }}
```

Comment:
Is this just skipping the test?

Reply:
For now, since `cudafe++` blows up the runner memory during compilation (this is also true on Perlmutter's nodes).